lib/sshutils/sftp/sftp.go

```diff
@@ -233,7 +231,7 @@ func expandPath(path string, getHomeDir homeDirRetriever) (string, error) {
 
 	// this is safe because we verified that all paths are non-empty
 	// in CreateUploadConfig/CreateDownloadConfig
-	return filepath.Join(homeDir, path[1:]), nil
+	return path.Join(homeDir, pathStr[1:]), nil
```

Comment:
I'd add a comment here, because most people are used to using filepath for file paths.

```
// note: intentionally prefer path.Join over filepath.Join because 
// SCP requires Linux-style path separators
```

Reply:
Added a comment to the `path` import as `path.Join` is used 3 other places in the file and it wouldn't make sense to only add a comment here

Reply:
The comments immediately above the package clause are the package level doc, which shows up for example when you run `go doc github.com/gravitational/teleport/lib/sshutils/sftp`.

I think this info is an internal detail and not relevant for the package level documentation.

Reply:
I tried to put the comment directly preceding the `path` import, but the `gci` linter didn't like that. Where do you think the next best place would be, directly under the `import` statement?

Reply:
Does `gci` mind if the comment is on the same line?

```
import (
  "path" // SCP requires Linux-style path separators
)
```

If not, that works. If that's still a problem, I don't think including the comment at all 3 call sites is a bad idea.

Reply:
Attempting the former here: https://github.com/gravitational/teleport/pull/21106